src/djlint/output.py

```diff
@@ -206,6 +201,14 @@ def build_stats_output(errors: List[Optional[Any]], config: Config) -> int:
         rule["rule"]["name"]: rule["rule"]["message"] for rule in config.linter_rules
     }
 
+    echo()
+    width, _ = shutil.get_terminal_size()
+    echo(
+        f"{Fore.GREEN}{Style.BRIGHT}Statistics{Style.RESET_ALL}\n{Style.DIM}"
+        + "".join(["─" for x in range(1, width)])
+        + Style.RESET_ALL
```

Comment:
In Python `'─' * 5` is '─────'
```suggestion
        f"{Fore.GREEN}{Style.BRIGHT}Statistics{Style.RESET_ALL}\n{Style.DIM}{}{'─' * width}{Style.RESET_ALL}"
```